### PR TITLE
fix: TT-290 added a new component (dropdown), and change the style the buttons associeted

### PR DESCRIPTION
**Pull request by kevinjlope**
**The current implementation is not very clear**

**I changed the style of the buttons**

### CODE REVIEW

File: src/app/modules/shared/components/dropdown/dropdown.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      btnIcon: 'fa-circle',
+      iconColor: 'text-danger'
+    };
+    component.info = fakeInfoActive;
+
+    component.changePropertiesItem(fakeInfoActive);
+
+    expect(expectChangeValue.btnName).toEqual('Inactive');
+    expect(fakeInfoActive.btnIcon).toEqual(expectChangeValue.btnIcon);
+    expect(expectChangeValue.iconColor).toEqual('text-danger');
```
Comment (jcalarcon98) [inline src/app/modules/shared/components/dropdown/dropdown.component.spec.ts:74]: Please add the suggestion spoken internally

File: src/app/modules/shared/components/dropdown/dropdown.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      btnIcon: 'fa-circle',
+      iconColor: 'text-success'
+    };
+    component.info = fakeInfoInactive;
+
+    component.changePropertiesItem(fakeInfoInactive);
+
+    expect(expectChangeValue.btnName).toEqual('Active');
+    expect(fakeInfoInactive.btnIcon).toEqual(expectChangeValue.btnIcon);
+    expect(expectChangeValue.iconColor).toEqual('text-success');
```
Comment (jcalarcon98) [inline src/app/modules/shared/components/dropdown/dropdown.component.spec.ts:103]: Same suggestion as above

File: src/app/modules/shared/components/dropdown/dropdown.component.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    key: '',
+    _status: false,
+    btnColor: '',
+    btnIcon: '',
+    btnIconTwo: '',
+    btnName: '',
+    iconColor: ''
+  };
+
+  changePropertiesItem(item: any){
```
Comment (jcalarcon98) [inline src/app/modules/shared/components/dropdown/dropdown.component.ts:29]: Maybe you can use object destructuring, like: 
```suggestion
  changePropertiesItem({ btnName, btnIcon} ){
```
Only if you consider necessary you can apply this change.

File: src/app/modules/shared/components/dropdown/dropdown.component.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    if (item.btnName === 'Active'){
+      this.dataChange.btnName = 'Inactive';
+      this.dataChange.btnIcon = item.btnIcon;
```
Comment (jcalarcon98) [inline src/app/modules/shared/components/dropdown/dropdown.component.ts:32]: I notice that this line:
```ts
this.dataChange.btnIcon = item.btnIcon;
```
is used in both -> if and else conditions.

As the condition is going to be the same, we can take it out even before/after the condition

**Review by jcalarcon98 — COMMENTED** [review]
Excellent work Kevin 💥 , please review the suggestions made and if you consider necessary you can implement them.

File: src/app/modules/shared/components/dropdown/dropdown.component.ts (hunk 4 above)
Comment (kevinjlope) [reply src/app/modules/shared/components/dropdown/dropdown.component.ts]: done

File: src/app/modules/shared/components/dropdown/dropdown.component.ts (hunk 3 above)
Comment (kevinjlope) [reply src/app/modules/shared/components/dropdown/dropdown.component.ts]: done

File: src/app/modules/shared/components/dropdown/dropdown.component.spec.ts (hunk 2 above)
Comment (kevinjlope) [reply src/app/modules/shared/components/dropdown/dropdown.component.spec.ts]: done

File: src/app/modules/shared/components/dropdown/dropdown.component.spec.ts (hunk 1 above)
Comment (kevinjlope) [reply src/app/modules/shared/components/dropdown/dropdown.component.spec.ts]: done

**Review by jcalarcon98 — APPROVED** [review]
Good Job! Kevin 💥

**Review by jase156 — CHANGES_REQUESTED** [review]

File: src/app/modules/activities-management/components/activity-list/activity-list.component.scss [hunk 5]
```diff
@@ -3,3 +3,12 @@
 .activity-list {
   overflow: auto;
 }
+.btn--switch{
```
[the remark this answers is not in the document]
Comment (kevinjlope) [reply src/app/modules/activities-management/components/activity-list/activity-list.component.scss]: review: https://blog.ida.cl/desarrollo/metodologia-bem-desarrollo-front-end/

File: src/app/modules/activities-management/components/activity-list/activity-list.component.scss [hunk 6]
```diff
@@ -3,3 +3,12 @@
...
+  font-size: 1rem;
```
[the remark this answers is not in the document]
Comment (kevinjlope) [reply src/app/modules/activities-management/components/activity-list/activity-list.component.scss]: the button component loses inheritance

File: src/app/modules/customer-management/components/projects/components/project-list/project-list.component.scss [hunk 7]
```diff
@@ -3,3 +3,13 @@
 .project-list {
   max-height: 300px; overflow: auto; display:inline-block; width: 100%;
 }
+
+.btn--switch{
+  font-size: 1rem;
+  border: 1px solid #c5c5c5;
```
[the remark this answers is not in the document]
Comment (kevinjlope) [reply src/app/modules/customer-management/components/projects/components/project-list/project-list.component.scss:9]: Is a good practice to create variables, but in this case this variable is not repeated.

File: src/app/modules/customer-management/components/projects/components/project-list/project-list.component.scss [hunk 8]
```diff
@@ -3,3 +3,13 @@
...
+  border-radius: 20px;
+}
+
+.iconTwo{
+  color: #a1a1a1;
```
[the remark this answers is not in the document]
Comment (kevinjlope) [reply src/app/modules/customer-management/components/projects/components/project-list/project-list.component.scss:14]: The same answer to above.

File: src/app/modules/activities-management/components/activity-list/activity-list.component.scss [hunk 9]
```diff
@@ -2,4 +2,6 @@
 
 .activity-list {
   overflow: auto;
+
 }
+
```
Comment (scastillo-jp) [inline src/app/modules/activities-management/components/activity-list/activity-list.component.scss:7]: delete this white line

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.spec.ts [hunk 10]
```diff
@@ -244,7 +248,9 @@ describe('CustomerTableListComponent', () => {
       _status: false,
       btnColor: 'btn-danger',
       btnIcon: 'fa-arrow-circle-down',
+      btnIconTwo:  'fa-caret-check',
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.spec.ts:251]: use only a space after `:`, replicate this change in the other parts with `btnIconTwo: 'fa-caret-check'`

File: src/app/modules/shared/components/dropdown/dropdown.component.scss [hunk 11]
```diff
@@ -0,0 +1,33 @@
+@import '../../../../../styles/colors.scss';
+
+.btn--switch {
+    font-size: 1rem;
+    border: 1px solid #c5c5c5;
```
Comment (scastillo-jp) [inline src/app/modules/shared/components/dropdown/dropdown.component.scss:5]: Import **#c5c5c5;** this color from `colors.scss`, you can use `lighten($primary-text, 30%)`

File: src/app/modules/shared/components/dropdown/dropdown.component.scss [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+    border-radius: 20px;
+    width: clamp(7rem, 50%, 12rem);
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+}
+
+.iconTwo {
+    color: #a1a1a1;
```
Comment (scastillo-jp) [inline src/app/modules/shared/components/dropdown/dropdown.component.scss:15]: Apply the same change

File: src/app/modules/customer-management/components/projects/components/project-list/project-list.component.scss (hunk 7 above)
[the remark this answers is not in the document]
Comment (kevinjlope) [reply src/app/modules/customer-management/components/projects/components/project-list/project-list.component.scss:9]: done!

File: src/app/modules/activities-management/components/activity-list/activity-list.component.scss [hunk 13]
```diff
@@ -2,4 +2,6 @@
 
 .activity-list {
   overflow: auto;
+
```
[the remark this answers is not in the document]
Comment (kevinjlope) [reply src/app/modules/activities-management/components/activity-list/activity-list.component.scss:5]: done

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.spec.ts (hunk 10 above)
Comment (kevinjlope) [reply src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.spec.ts:251]: done

File: src/app/modules/shared/components/dropdown/dropdown.component.scss (hunk 11 above)
Comment (kevinjlope) [reply src/app/modules/shared/components/dropdown/dropdown.component.scss:5]: done

File: src/app/modules/shared/components/dropdown/dropdown.component.scss (hunk 12 above)
Comment (kevinjlope) [reply src/app/modules/shared/components/dropdown/dropdown.component.scss:15]: done

**Review by scastillo-jp — APPROVED** [review]

**Review by edgardavid2015 — APPROVED** [review]
I already check this new features, and for me it works fine. The only observation that I have is in the word "associeted", is misspelled.

**Pull request merged**